deploy/manager/deployments.yaml

```diff
@@ -1,88 +1,61 @@
----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: system
----
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: scylla-manager
+  namespace: scylla-manager
 spec:
+  replicas: 1
   selector:
     matchLabels:
       app: scylla-manager
-  replicas: 1
   strategy:
     type: Recreate
   template:
     metadata:
       labels:
         app: scylla-manager
     spec:
-      volumes:
-        - name: scylla-manager-config-volume
-          configMap:
-            name: manager-config
       containers:
-        - name: scylla-manager
-          image: manager:latest
-          imagePullPolicy: IfNotPresent
+        - args:
+            - --config-file=/mnt/etc/scylla-manager/scylla-manager.yaml
           command:
             - /usr/bin/scylla-manager
-          args:
-            - --config-file=/mnt/etc/scylla-manager/scylla-manager.yaml
           env:
             - name: POD_NAMESPACE
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+          image: scylladb/scylla-manager:2.2.0
+          imagePullPolicy: IfNotPresent
```

Comment:
nit: I'd drop the pull policy, it's usually enforced or defaulted at cluster level